index.html

```diff
@@ -10,6 +10,12 @@
   <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no,user-scalable=0">
   <link rel="stylesheet" href="style.css">
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="/favicon.png"/>
+  <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
+  <link rel="icon" href="/images/favicon-16.png" sizes="16x16">
```

Comment:
```suggestion
  <link rel="icon" href="/assets/favicon-16.png" sizes="16x16">
```

this is the actual path. please correct everywhere.